subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,28 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+        cid = self.get_cid()
+        self.sub_stuff['metadata'] = (
+            self.sub_stuff['cont'].json_by_long_id(cid))
```

Comment:
This is fine here, though for more complex/long cases, keep Ed's suggestion in mind:  

> Using a temporary variable to avoid line wrap is fine

Names and references in python (generally) tend to be on the long-side (compared to other languages).  It's totally cool to use other temporaries to hold values that are immediately re-assigned.  Python will most likely just optimize them away, but they do add a bit to readability.  Really long call references can be shortened in a similar way, e.g.:

```python
_blah = self.other.some['really'].long.reference['to']['a'].method_or_function
result = _blah(parameters)
```